packages/opentelemetry-sdk-trace-base/src/export/InMemorySpanExporter.ts

```diff
@@ -47,6 +47,14 @@ export class InMemorySpanExporter implements SpanExporter {
 
   shutdown(): Promise<void> {
     this._stopped = true;
+    return this._flush();
+  }
+
+  forceFlush(): Promise<void> {
+    return this._flush();
+  }
+
+  private _flush(): Promise<void> {
     this._finishedSpans = [];
```

Comment:
Flushing should not reset. There is a dedicated method for that

Reply:
@dyladan thank you for the review, can you give your thoughts on BatchSpanProcessorBase (I posted a comment above) should we reimplement it, or leave as is